rpc/internal/sps.service.go

```diff
@@ -0,0 +1,74 @@
+package internal
+
+import (
+	"time"
+
+	"github.com/evergreen-ci/poplar"
+)
+
+type TaskInformation struct {
+	Project   string `json:"project"`
+	Version   string `json:"version"`
+	Variant   string `json:"variant"`
+	Order     int    `json:"order"`
+	TaskName  string `json:"task_name"`
+	TaskId    string `json:"task_id"`
+	Execution int    `json:"execution"`
+	Mainline  bool   `json:"mainline"`
+}
+
+type TestInformation struct {
+	TestName  string           `json:"test_name"`
+	Arguments map[string]int32 `json:"arguments"`
+}
+
+type TestMetric struct {
+	Name  string      `json:"name"`
```

Comment:
Version/type aren't here, but I think I remember that being intentional?

Reply:
Yes, that's intentional. Those don't seem to be used by anything (except for the rollup code that's no longer in use). We give defaults to these in the SPS code